cores/esp8266/Esp.cpp

```diff
@@ -171,6 +171,11 @@ uint32_t EspClass::getFreeHeap(void)
     return system_get_free_heap_size();
 }
 
+uint16_t EspClass::getMaxFreeBlockSize(void)
+{
+    return system_get_free_heap_size();
```

Comment:
What's the difference between this and the return via maxBlock pointer in getHeapFragmentation?

Reply:
This is a mistake :) (the example uses getHeapFragmentation parameters)